indexd/single_table_settings.py

```diff
@@ -0,0 +1,69 @@
+import os
+
+from indexd.index.drivers.single_table_alchemy import SingleTableSQLAlchemyIndexDriver
```

Comment:
Most of `single_table_settings` is a copy/merge of `default_settings` and `default_test_settings` - I notice a few changes but not sure why they're needed, for example `DIST` is different and `DRS_SERVICE_INFO.id` is not in all the settings files

Is adding a new file necessary? Can't we switch between `SQLAlchemyIndexDriver` and `SingleTableSQLAlchemyIndexDriver` depending on another config field, or even an env var?

If that's not possible, can we remove the duplicated settings and just overwrite `default_settings` with whatever is in `single_table_settings`?
Maybe `single_table_settings` could do `from indexd.default_settings import *` like `default_test_settings` does, and just set a different `CONFIG["INDEX"]`

Reply:
Removing the new file and consolidating everything in conftest